tasks/Module 4/task_6.py

```diff
@@ -0,0 +1,26 @@
+"""Task 6 of Module 4."""
+from tkinter.filedialog import askopenfilename
+from pathlib import Path
+import os
+import PyPDF2
+
+
+def brute_force_pdf():
+    """Brute force encrypted PDF file with provided list of words."""
+    pdf_file = askopenfilename(title='Select encrypted PDF file')
+    pwd_file_path = Path(pdf_file)
+    os.chdir(pwd_file_path.parent)
+
+    dict_file = askopenfilename(title='Select encrypted Dictionary file')
+    dict_file_path = Path(dict_file)
+
+    with open(dict_file_path, "r") as dict_file_content:
```

Comment:
If you like you can also open multiple files in one `with` statement:
```python
with open(dict_file_path, "r") as dict_file_content, open(pwd_file_path.name, 'rb') as pdf_content:
```

Reply:
The code is changed based on above